app/experimenter/nimbus-ui/src/components/Summary/index.tsx

```diff
@@ -29,9 +30,34 @@ const Summary = ({ experiment }: SummaryProps) => {
 
   return (
     <div data-testid="summary">
-      <h2 className="h5 mb-3">Timeline</h2>
+      <h2 className="h5 mb-3">
+        Timeline
+        {status.live && (
+          <>
+            {experiment.isEndRequested && (
+              <StatusPill
+                testId="pill-end-requested"
+                label="Experiment End Requested"
+              />
+            )}
+            {experiment.isEnrollmentPaused === false && (
+              <StatusPill
+                testId="pill-enrolling-active"
+                label="Enrolling Users in Progress"
+              />
+            )}
+            {experiment.isEnrollmentPaused && experiment.enrollmentEndDate && (
+              <StatusPill
+                testId="pill-enrolling-complete"
+                label="Enrollment Complete"
+              />
+            )}
+          </>
+        )}
+      </h2>
```

Comment:
This might be worth encapsulating as its own sub-component? Not crucial though.

Reply:
Works for me!